### ISSUE TITLE
[$20] [MSFT-193] [Registration]: Links should be underlined

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2859]
**Steps to Reproduce**
1. Launch https://accounts.topcoder.com/member/registration 
2. Check the available links in the page

**Expected Result**
Links should be underlined

**Actual Result**
Links are NOT underlined

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FLinks%20are%20NOT%20underlined

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096261 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096261 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096261 has been updated - it has been assigned to marioskranitsas.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by marioskranitsas** [comment]
PR https://github.com/topcoder-platform/community-app/issues/2859

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2859 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2859 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2859 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097083 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097083 has been updated - it has been assigned to marioskranitsas.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: macOS
Browser: Chrome
URL : https://accounts.topcoder-dev.com/member/registration

**Comment by nithyaasworld** [comment]
@drasticdpk @marioskranitsas "Or Login with" which is not a link, is also underlined.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097083 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@crazyk07 @veshu @nithyaasworld  PR https://github.com/appirio-tech/accounts-app/pull/282

**Comment by lakshmiathreya** [comment]
Verified with Win 10 Chrome NVDA & Mac OS Chrome Voice Over - Test url.
-- 'Terms' and 'Privacy Policy' are links but not underlined. 
-- 'PRIVACY POLICY' at the end of the page is not underlined. 
-- 'LOG IN' is a link and is underlined.

**Comment by rashmi73** [comment]
@lakshmiathreya kindly find attach screenshot which is taken from my PR which is merged, also verified on qa-accessibility branch.
it is perfect.

![2859](https://user-images.githubusercontent.com/41687423/62253762-a01b4c80-b414-11e9-8741-492abee8cf0e.png)


can you please confirm once again? because everything is underlined.

**Comment by mishacucicea** [comment]
@nkumar-topcoder This is working in the `qa-accessibility` branch. Was this deployed on dev environment?

**Comment by tosha5252** [comment]
This works in the dev end

Win10/Chrome

**Comment by tosha5252** [comment]
'Terms' and 'Privacy Policy' are not underlined in prod env

Win10/Chrome/Edge

[ref video.zip](https://github.com/topcoder-platform/community-app/files/3500964/ref.video.zip)

**Comment by nkumar-topcoder** [comment]
@mishacucicea Can you please reverify master and dev branch for what's missing underline code.

is rashmi73 is still available else we can assign to @r0hit-gupta

**Comment by mishacucicea** [comment]
@nkumar-topcoder The fix in on `qa-accessibility`, file [register.scss](https://github.com/appirio-tech/accounts-app/blob/qa-accessibility/app/styles/tc/register.scss) lines 5-10.
![image](https://user-images.githubusercontent.com/1721199/63120209-851b1180-bfaa-11e9-8c7b-add2d3992319.png)
Can you get those changes in? They might of been part of other commits and seems they got lost.

**Comment by rashmi73** [comment]
yes i am available

**Comment by nkumar-topcoder** [comment]
thanks @mishacucicea I merged. @tosha5252 can you pls verify at DEV. 
@rashmi73 this was a merge issue, hence pls ignore but thanks for letting know your availability.

**Comment by tosha5252** [comment]
Verified working at DEV

WIn10/Chrome

**Comment by nkumar-topcoder** [comment]
@tosha5252 Can you please verify in prod.

**Comment by drasticdpk** [comment]
@tosha5252 @nkumar-topcoder  now it is working as expected.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097083<br/><br/>```This is an automated message for crazyk via Topcoder X```
